serverlessworkflow/modules/ROOT/pages/getting-started/create-your-first-workflow-service.adoc

```diff
@@ -1 +1,257 @@
+and onwards to create the application step by step.
+
+However, you can go right to the completed example by cloning the git repository:
+
+The solution is located in the `kogito-quarkus-examples/serverless-workflow-hello-world`
+https://github.com/kiegroup/kogito-examples/tree/main/kogito-quarkus-examples/serverless-workflow-hello-world[directory].
+
+[[boostrapping-the-project]]
+== Bootstrapping the Project
+
+Run the following command in a terminal window to create the project skeleton:
+
+.Using Quarkus CLI
+[source,shell]
+----
+quarkus create app \
+    -x=kogito-quarkus-serverless-workflow \
+    -x=quarkus-container-image-jib \
+    -x=quarkus-resteasy-jackson \
+    -x=quarkus-smallrye-openapi \
+    org.kie.kogito.examples:serverless-workflow-hello-world:1.0
+----
+
+.Using Maven
+[source,shell,subs="attributes"]
+----
+mvn io.quarkus.platform:quarkus-maven-plugin:{quarkus_version}:create \
+    -DprojectGroupId=org.kie.kogito.examples \
+    -DprojectArtifactId=serverless-workflow-hello-world \
+    -Dextensions="kogito-quarkus-serverless-workflow,quarkus-container-image-jib,quarkus-resteasy-jackson,quarkus-smallrye-openapi"
+cd getting-started
+----
+
+`org.kie.kogito.examples`, `serverless-workflow-hello-world` and `1.0` are respectively the group id, artifact id,
+and version of your project.
+
+This command will create a Maven Quarkus project in the `serverless-workflow-hello-world` directory with all the
+required Kogito dependencies.
+
+* `kogito-quarkus-serverless-workflow` - Adds support for Serverless Workflow
+* `quarkus-container-image-jib` - Adds support for Container Image Builds
+* `quarkus-resteasy-jackson` - Adds support for RESTEasy, which is required by the generated REST
+resources that are used to start the flow process through an HTTP request.
+* `quarkus-smallrye-openapi` - Adds support for Swagger documentation.
+The SwaggerUI will be available at `http://localhost:8080/q/swagger-ui/` when you run the application.
+
+Next, to make sure everything is working fine, try to compile the project with:
+
+.Using Quarkus CLI
+[source,shell]
+----
+quarkus build
+----
+
+.Using Maven
+[source,shell]
+----
+mvn clean package
+----
+
+=== Creating Your First Workflow
+
+Create a file named `hello.sw.json` in the `src/main/resources` directory with the following content:
+
+[source,json]
+----
+{
+  "id": "hello_world",
+  "version": "1.0",
+  "specVersion": "0.8",
+  "name": "Hello World Workflow",
+  "description": "JSON based hello world workflow",
+  "start": "Inject Hello World",
+  "states": [
+    {
+      "name": "Inject Hello World",
+      "type": "inject",
+      "data": {
+        "greeting": "Hello World"
+      },
+      "transition": "Inject Mantra"
+    },
+    {
+      "name": "Inject Mantra",
+      "type": "inject",
+      "data": {
+        "mantra": "Serverless Workflow is awesome!"
+      },
+      "end": true
+    }
+  ]
+}
+----
```

Comment:
I believe the workflow definition would deserve some description and guidance on how to read it. That, workflow definition is uniquely identified by id and generated REST endpoints will be based on that. Also that the workflow contains 2 states. The one where workflow starts (start) is "Inject Hello World" of the type Inject - perhaps add some definition of Inject state from the spec what it does - in our case it sets greeting field name in workflow data. Then transitions to "Inject Mantra" where it sets "mantra" field name in workflow data and that's where the workflow ends. The result/output of workflow is the workflow data. --- or something like that.